cmd/infracost/output.go

```diff
@@ -143,6 +143,33 @@ func outputCmd(ctx *config.RunContext) *cobra.Command {
 			if cmd.Flags().Changed("fields") && !contains(validFieldsFormats, format) {
 				ui.PrintWarning(cmd.ErrOrStderr(), "fields is only supported for table and html output formats")
 			}
+
+			if ctx.Config.EnableDashboard {
+				if ctx.Config.IsSelfHosted() {
+					ui.PrintWarning(cmd.ErrOrStderr(), "The dashboard is part of Infracost's hosted services. Contact hello@infracost.io for help.")
+				}
+
+				projectContexts := []*config.ProjectContext{}
+				for range combined.Projects {
+					projectContexts = append(projectContexts, config.EmptyProjectContext())
+				}
```

Comment:
would prefer allocating the length of the slice since we know it's length 
```suggestion
				projectContexts := make([]*config.ProjectContext{}, len(combined.Projects)
				for i, _ := range combined.Projects {
					projectContexts[i] = config.EmptyProjectContext()
				}
```